config/services.yaml

```
@@ -6,6 +6,78 @@
+      - { name: 'api_platform.filter', id: 'customer.mongodb.order' }
+
+  app.customer.mongodb.range_filter:
+    parent: 'api_platform.doctrine_mongodb.odm.range_filter'
+    class: App\Shared\Infrastructure\Filter\UlidRangeFilter
+    arguments: [{ 'ulid': ~ }]
+    tags:
+      - { name: 'api_platform.filter', id: 'mongodb.range' }
+
+  app.customer.mongodb.date_filter:
+    parent: 'api_platform.doctrine_mongodb.odm.date_filter'
+    arguments: [{ 'createdAt': ~, 'updatedAt': ~ }]
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)